gazebo_ros/include/gazebo_ros/node.hpp

```diff
@@ -129,6 +130,9 @@ class GAZEBO_ROS_NODE_PUBLIC Node : public rclcpp::Node
   /// Inherit constructor
   using rclcpp::Node::Node;
 
+  // A handler for the param change callback.
+  rclcpp::node_interfaces::OnSetParametersCallbackHandle::SharedPtr param_change_callback_handler;
```

Comment:
style nit: Use an underscore suffix like the other member variables
```suggestion
  rclcpp::node_interfaces::OnSetParametersCallbackHandle::SharedPtr param_change_callback_handler_;
```